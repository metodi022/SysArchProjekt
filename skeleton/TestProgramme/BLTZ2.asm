lui $1, 1
bltz $1, end
ori $1, 3
end:
ori $1, 4