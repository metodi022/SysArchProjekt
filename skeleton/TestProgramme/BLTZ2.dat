0x3c010001
0x04200001
0x34210003
0x34210004
