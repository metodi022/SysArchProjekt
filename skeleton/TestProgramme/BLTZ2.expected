@1 00010007